polaris/polaris/utils.py

```diff
@@ -135,24 +135,23 @@ def create_stellar_deposit(
         transaction.save()
         raise ValueError(transaction.status_message)
 
-    # if we don't know if the destination account exists
-    if not destination_exists:
```

Comment:
So you're right we don't need to check this because the destination account should always exist.

But that means `destination_exists` doesn't even need to be a parameter, and all invocations of `create_stellar_deposit()` with that parameter should be adjusted.